src/plugins/typingIndicator/index.tsx

```diff
@@ -47,14 +47,18 @@ function TypingIndicator({ channelId }: { channelId: string; }) {
             return oldKeys.length === currentKeys.length && currentKeys.every(key => old[key] != null);
         }
     );
-
+    const id: string = useStateFromStores([SelectedChannelStore], () => SelectedChannelStore.getChannelId());
```

Comment:
```suggestion
    const currentChannelId: string = useStateFromStores([SelectedChannelStore], () => SelectedChannelStore.getChannelId());
```

I did an oopsie my bad

Reply:
Nah, that one was on me. I saw a suggested change from a contributor to this project and went "LGTM 👍" and committed without actually building the changes to review it